### DISCUSSION TITLE
asset maintenance & updates

### DISCUSSION THREAD

**Discussion by tylervick** [XCTestHTMLReport/XCTestHTMLReport#246] [General]
The process of adding/updating assets is not documented. For example, we do not have an asset for the "Mixed" test status to use as a badge in generated HTML.

Our existing badges/other assets are currently inlined with global CSS.

**Comment by tylervick** [comment]
@nacho4d do you know how we've historically gathered existing assets? I see PR #172 you implemented with "Skipped" tests included the relevant badge.

I'd like to update these, ideally filling out any missing assets and avoiding any use of copyright IP.

**Reply by nacho4d** [reply]
Hello Tyler,

Sorry I do not know. 

Current assets don't look to be copied from old Xcode. They seem to created by someone else. There are some edges I see a difference. Also, when I implemented "Skipped" I created the image myself.

**Reply by tylervick** [reply]
Thanks! That's what I needed to know :) 
I can fire up Figma and take a crack at a new "Mixed" result icon.
